demo/js/demo.js

```diff
@@ -208,12 +216,16 @@ function initDemo() {
 
   toggleExtrusion.addEventListener('click', function () {
     changeRenderExtrusion(!!toggleExtrusion.checked);
+    preview.initScene();
     preview.render();
   });
```

Comment:
`preview.render()` calls `initScene`, do you need to call it here?

Reply:
good catch!